### ISSUE TITLE
FileTransferServer.packet_CLOSE raises AttributeError for non-open handle

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10293]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#10293|
|Type|enhancement|
|Created|2022-01-21 21:37:41Z|

When FileTransferServer tries to process a CLOSE request with a handle identifier that is not open it tries to call `self._ebClose`.  There is no such attribute so this raises an `AttributeError` and never responds to the request.

<details><summary>Searchable metadata</summary>

```
trac-id__10293 10293
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1642801061249338 1642801061249338
changetime__1643039762895677 1643039762895677
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

OpenSSH (if I am even looking at the right codebase) seems to return ENOENT in this case:

https://github.com/openssh/openssh-portable/blob/68085066b6bad43643b43f5957fcc5fd34782ccd/sftp-server.c#L447-L464

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

https://github.com/twisted/twisted/pull/1685

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_4"></a>|@wsanchez set status to `closed`|
|-|-|

reviewed

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set status to `reopened`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @exarkun<br>@wsanchez set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

https://github.com/twisted/twisted/pull/1685 merged in a033c848d3adc1359d88a9806db8bbd427313841

I followed the guidance for merge commit message but this ticket did not auto-close.
